test/EventHandlers/CLPool/CLPool.test.ts

```diff
@@ -192,9 +198,8 @@ describe("CLPool Event Handlers", () => {
   });
 
   describe("Collect Event", () => {
-    let mockEvent: EventFunctions_mockEventData;
-    let mockEventData: EventFunctions_mockEventData;
-    let setupDB: MockDb;
+    let mockEvent: CLPool_Collect_event;
+    let setupDB: TestHelpers_MockDb_t;
```

Comment:
revisit

Reply:
I believe it's related to the `setupDB`, like it wasn't clear to me why it's not just `const setupDB = MockDb.createMockDb()` so I wanted to revisit it.

Reply:
I think code was declaring `const MockDb = MockDb.createMockDb()` above to be re-use `MockDb` for other `setupDB` later in the test 👍 

relatively minor difference - if you agree, we can resolve this comment and make this PR ready for review!